fhttp/http.go

```diff
@@ -1147,14 +1148,15 @@ func DebugHandler(w http.ResponseWriter, r *http.Request) {
 // Serve starts a debug / echo http server on the given port.
 // TODO: make it work for port 0 and return the port found and also
 // add a non blocking mode that makes sure the socket exists before returning
-func Serve(port int, debugPath string) {
+func Serve(port string, debugPath string) {
 	startTime = time.Now()
-	fmt.Printf("Fortio %s echo server listening on port %v\n", periodic.Version, port)
 	if debugPath != "" {
 		http.HandleFunc(debugPath, DebugHandler)
 	}
 	http.HandleFunc("/", EchoHandler)
-	if err := http.ListenAndServe(fmt.Sprintf(":%d", port), nil); err != nil {
+	nPort := fnet.NormalizePort(port)
+	if err := http.ListenAndServe(nPort, nil); err != nil {
```

Comment:
I think this doesn't return if I call - can you check that you see the output when you run  `fortio server`

Reply:
You're right. I apologize for not catching that in my testing. I will update and resubmit my PR.